integration-tests/api-gateway/aws/env-variables.ts

```diff
@@ -0,0 +1,48 @@
+export const PRIVATE_API = "6glrjvv0ei"
+export const PUBLIC_API = "zaj2n1sknd"
```

Comment:
remove this but instead rely on the stack outputs, the your can use better names for 
  privateApi = `${preOutput.PrivateApiGatewayId}`;
  publicApi = `${preOutput.PublicApiGatewayId}`;